backend/src/main/java/pl/edu/agh/awiteks_backend/configs/WebMVCConfiguration.java

```diff
@@ -0,0 +1,22 @@
+package pl.edu.agh.awiteks_backend.configs;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebMVCConfiguration implements WebMvcConfigurer {
+
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
+        // TODO get this from env file or inject from elsewhere
+        final String[] allowedOrigins = {
+                "http://localhost:3000", // React dev server
+                "http://127.0.0.1:3000"  // probably not needed
+        };
```

Comment:
Można zrobić coś takiego, że w pliku `application.properties` napiszesz np. coś takiego:
```
react.dev.server.url=${REACT_DEV_SERVER_URL:#{'http://localhost:3000}}
```
W tym momencie, jeżeli nic nie będzie ustawione w zmiennej środowiskowej `REACT_DEV_SERVER_URL` to weźmie domyślny adres, czyli `localhost:3000`.

Teraz wystarczy tylko wstrzyknąć tą zmienną do konfiguracji:
```java
@Value("${react.dev.server.url}")
private String reactDevServerUrl;
```

Reply:
Zmieniłem zgodnie z sugestią.